roles/set_openstack_containers/tasks/main.yml

```diff
@@ -91,7 +91,9 @@
   ansible.builtin.include_role:
     name: 'install_yamls_makes'
     tasks_from: 'make_openstack_wait'
-  when: operator_name == 'openstack'
+  when: 
```

Comment:
I think we can replace this wait with:
```
- name: Wait for reinstallation of meta operator
  environment:
    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
    PATH: "{{ cifmw_path }}"
  ansible.builtin.shell:
    cmd: >-
      oc get csv -l operators.coreos.com/openstack-operator.openstack-operators
      --namespace={{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }}
      --no-headers=true | grep -i "succeeded"
  register: operator_status
  until: operator_status.rc == 0
  changed_when: false
  retries: 30
  delay: 30 
 when: operator_name == 'openstack'
```
This is basically the same what openstack_wait does.

Reply:
Hum... the operator.coreos.com is retrieved from https://github.com/openstack-k8s-operators/install_yamls/blob/main/scripts/get-operator-status.sh#L12 :

```
oc get csv -n ${OPERATOR_NAMESPACE} -o jsonpath='{range .items[*]}{@.metadata.name}{"\n"}{end}' | grep -E "^${OPERATOR_NAME}-operator\.v"
```
so maybe we should use that also ... but it's unwrapping the script ... if the makefile target wasn't dependent on openstack target ...

Oki, I'll test with both of them, included.

Reply:
openstack-operator is the name of operator retrieved by get-operator-status , it might or not change in future
You can implement two tasks, one for revival of exact name and second for waiting for reinstallation